### PR TITLE
Update intro.adoc

### PR DESCRIPTION
**Pull request by nahern**
folding in some copy from the testnet website.

### CODE REVIEW

File: plutus-tutorial/doc/intro.adoc [hunk 1]
```diff
@@ -29,3 +29,14 @@ authors do not write Plutus Core directly. The Plutus Platform is a
 software development kit to enable smart contract authors to easily
 write smart contracts, including the logic that will eventually be run
 on the blockchain as Plutus Core.
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground editor, or you can use your preferred code editor and save the 
+files in Haskell format (.hs) to get the syntax highlighting,
```
Comment (michaelpj) [inline plutus-tutorial/doc/intro.adoc]: Use code formatting for `.hs`.

File: plutus-tutorial/doc/intro.adoc [hunk 2]
```diff
@@ -29,3 +29,14 @@ authors do not write Plutus Core directly. The Plutus Platform is a
 software development kit to enable smart contract authors to easily
 write smart contracts, including the logic that will eventually be run
 on the blockchain as Plutus Core.
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground editor, or you can use your preferred code editor and save the
```
Comment (michaelpj) [inline plutus-tutorial/doc/intro.adoc]: This is the first mention of Plutus Playground in the document, it probably needs an introduction. And you probably want to link to it - there's an attribute for the url, see other places in the document where we link to it.

**Comment by michaelpj** [comment]
Ping.

File: plutus-tutorial/doc/intro.adoc (hunk 2 above)
Comment (nahern) [reply plutus-tutorial/doc/intro.adoc]: will include description and link

File: plutus-tutorial/doc/intro.adoc (hunk 1 above)
Comment (nahern) [reply plutus-tutorial/doc/intro.adoc]: do you mean replace this sentence? unsure as to what that means

Comment (nahern) [reply plutus-tutorial/doc/intro.adoc]: will do

File: plutus-tutorial/doc/intro.adoc [hunk 3]
```diff
@@ -29,3 +29,20 @@ authors do not write Plutus Core directly. The Plutus Platform is a
 software development kit to enable smart contract authors to easily
 write smart contracts, including the logic that will eventually be run
 on the blockchain as Plutus Core.
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground (https://prod.playground.plutus.iohkdev.io/).
```
Comment (michaelpj) [inline plutus-tutorial/doc/intro.adoc]: Please use the attribute like I suggested: `{playground}[Plutus Playground]`

File: plutus-tutorial/doc/intro.adoc [hunk 4]
```diff
@@ -29,3 +29,20 @@ authors do not write Plutus Core directly. The Plutus Platform is a
...
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground (https://prod.playground.plutus.iohkdev.io/). 
+Plutus Playground is a lightweight, web-based environment 
+for exploratory Plutus development. Here you can easily write and deploy your 
+contracts without the overhead of installing and maintaining a full development 
+environment and blockchain. 
+
+You can also use your preferred code editor and save the 
+files in Haskell format `(.hs)` to get the syntax highlighting,
```
Comment (michaelpj) [inline plutus-tutorial/doc/intro.adoc]: It's only the extension that's code, the parentheses are part of the text, so ```(`.hs`)```

**Comment by michaelpj** [comment]
Ping.

**Pull request merged**